testutil/manifest_app.go

```diff
@@ -0,0 +1,58 @@
+package testutil
+
+import (
+	"testing"
+
+	"github.com/ovrclk/akash/manifest"
+	"github.com/ovrclk/akash/types"
+	"github.com/ovrclk/akash/types/unit"
+)
+
+// A generator that represents a real-world, deployable configuration.
+
+type manifestGeneratorApp struct{}
+
+func (mg manifestGeneratorApp) Manifest(t testing.TB) manifest.Manifest {
```

Comment:
they do so that it implements [`testutil.ManifestGenerator`](https://github.com/ovrclk/akash/pull/644/files#diff-ba687b123baac8c17c7512ffeeb17550R18).

Reply:
ahh I see